quipucords/api/scantask/model.py

```diff
@@ -213,7 +212,7 @@ def log_message(
         """Log a message for this task."""
         # pylint: disable=no-member
         if exception:
-            logger.error(traceback.format_tb(exception.__traceback__))
+            logger.exception("Got an exception.")
```

Comment:
Won't we lose what the exception is, in addition to removing the traceback? Or does `logger.exception` handle that?

Reply:
logger.exception handles that =D
turns out that formatting in the old way was losing a lot of context from the traceback.

This is how it looks like now
![image](https://user-images.githubusercontent.com/6073215/199998960-e2960906-1892-4b6e-b270-f8ab214725c7.png)